apps/web/src/components/Basenames/RegistrationForm/index.tsx

```diff
@@ -341,7 +311,7 @@ export default function RegistrationForm() {
             </div>
           )}
 
-          {registerNameError && (
+          {!!registerNameError && (
```

Comment:
this resolves typescript error: `Type 'unknown' is not assignable to type 'ReactNode'.